modules/tests/support/model_based/IBC.cfg

```diff
@@ -0,0 +1,12 @@
+CONSTANTS
+    ChainIds = {"chain-A", "chain-B"}
```

Comment:
Would be a good idea to give chain identifiers the same format as in production, e.g., "chainA-0" and "chainB-0".

Reply:
Could you point me to place where this format is defined? Can there be a `"chainA-0"` and a `"chainA-1"`?

Reply:
This format is just a convention at the moment, not sure it's formally defined anywhere.

> Can there be a "chainA-0" and a "chainA-1"?

This would be the same chain, in two subsequent epochs or revisions: revision `0` (due to suffix `-0`), and revision `1`. This also explains why a ibc::Height has two fields: a revision and a number.

For "chainA-0", the revision would be "0" at all height numbers of that chain.
For "chainA-1", the revision would be "1" at all height numbers of that chain...

Reply:
That makes sense, thanks! Done in 5d6bf08c7fcfe4d9040c661f0f044ebf641ea31f.